src/languages/julia.js

```diff
@@ -2,95 +2,328 @@
 Language: Julia
 Description: Julia is a high-level, high-performance, dynamic programming language.
 Author: Kenta Sato <bicycle1885@gmail.com>
-Contributors: Alex Arslan <ararslan@comcast.net>
+Contributors: Alex Arslan <ararslan@comcast.net>, Fredrik Ekre <ekrefredrik@gmail.com>
 Website: https://julialang.org
 */
 
 export default function(hljs) {
   // Since there are numerous special names in Julia, it is too much trouble
   // to maintain them by hand. Hence these names (i.e. keywords, literals and
-  // built-ins) are automatically generated from Julia v0.6 itself through
+  // built-ins) are automatically generated from Julia 1.5.2 itself through
   // the following scripts for each.
 
-  // ref: http://julia.readthedocs.org/en/latest/manual/variables/#allowed-variable-names
+  // ref: https://docs.julialang.org/en/v1/manual/variables/#Allowed-Variable-Names
   var VARIABLE_NAME_RE = '[A-Za-z_\\u00A1-\\uFFFF][A-Za-z_0-9\\u00A1-\\uFFFF]*';
 
+  // # keyword generator, multi-word keywords handled manually below (Julia 1.5.2)
+  // import REPL.REPLCompletions
+  // foreach(x -> println("\'", x, "\',"), ["in", "isa", "where"])
+  // for kw in collect(x.keyword for x in REPLCompletions.complete_keyword(""))
+  //     if !(contains(kw, " ") || kw == "struct")
+  //         println("\'", kw, "\',")
+  //     end
+  // end
+  var KEYWORD_LIST = [
+    'baremodule',
+    'begin',
+    'break',
+    'catch',
+    'ccall',
+    'const',
+    'continue',
+    'do',
+    'else',
+    'elseif',
+    'end',
+    'export',
+    'false',
+    'finally',
+    'for',
+    'function',
+    'global',
+    'if',
+    'import',
+    'let',
+    'local',
+    'macro',
+    'module',
+    'quote',
+    'return',
+    'true',
+    'try',
+    'using',
+    'while',
+  ]
+
+  // # literal generator (Julia 1.5.2)
+  // import REPL.REPLCompletions
+  // foreach(x -> println("\'", x, "\',"), ["true", "false"])
+  // for compl in filter!(x -> isa(x, REPLCompletions.ModuleCompletion) && (x.parent === Base || x.parent === Core),
+  //                     REPLCompletions.completions("", 0)[1])
+  //     try
+  //         v = eval(Symbol(compl.mod))
+  //         if !(v isa Function || v isa Type || v isa TypeVar || v isa Module || v isa Colon)
+  //             println("\'", compl.mod, "\',")
+  //         end
+  //     catch e
+  //     end
+  // end
+  var LITERAL_LIST = [
+    'ARGS',
+    'C_NULL',
+    'DEPOT_PATH',
+    'ENDIAN_BOM',
+    'ENV',
+    'Inf',
+    'Inf16',
+    'Inf32',
+    'Inf64',
+    'InsertionSort',
+    'LOAD_PATH',
+    'MergeSort',
+    'NaN',
+    'NaN16',
+    'NaN32',
+    'NaN64',
+    'PROGRAM_FILE',
+    'QuickSort',
+    'RoundDown',
+    'RoundFromZero',
+    'RoundNearest',
+    'RoundNearestTiesAway',
+    'RoundNearestTiesUp',
+    'RoundToZero',
+    'RoundUp',
+    'VERSION',
```

Comment:
```suggestion
    'VERSION|0',
```

Reply:
Sees like this could be found in almost anything, to lowering it's relevance.

Reply:
Alright, this is for autodetection? Isn't this needed for e.g. `for` which exists in almost any language?